clients/wallet/src/cache.rs

```diff
@@ -21,42 +24,102 @@ macro_rules! unwrap_or_return {
 	};
 }
 
-/// Contains the path where the transaction envelope will be saved.
 #[derive(Debug, Clone)]
-pub struct TxEnvelopeStorage {
+pub struct Cache {
```

Comment:
@b-yap 
I would appreciate if you rename it from Cache to something more specific or re-export it with other, more specific name.

Reply:
@RustNinja this cache serves as a storage/backup for the wallet's **_pagination_** and _**tx envelopes**_. And who knows in the future, it could be for something else as well. 

is it ok if I rename it to "WalletStateStorage" ? Because it saves the ff. info in a file:
1. at what cursor/page the latest transactions will be, 
2. the pending transactions of the wallet

@ebma  do you have any ideas for the name?

Reply:
see [a9b9113](https://github.com/pendulum-chain/spacewalk/pull/335/commits/a9b9113a30bf3aef95bba8c45f09e32ef4909d1d)

Reply:
I think 'WalletStateStorage' is a fitting name 👍